src/common/entity/binary_sensor_icon.ts

```diff
@@ -7,6 +7,8 @@ export const binarySensorIcon = (state: HassEntity) => {
   switch (state.attributes.device_class) {
     case "battery":
       return activated ? "hass:battery" : "hass:battery-outline";
+    case "battery_charging":
+      return activated ? "hass:flash" : "hass:flash-off";
```

Comment:
`battery-charging` / `battery` ?

Reply:
According to the [binary_sensor](https://github.com/home-assistant/core/blob/dev/homeassistant/components/binary_sensor/__init__.py):
```python
# On means low, Off means normal
DEVICE_CLASS_BATTERY = "battery"

# On means charging, Off means not charging
DEVICE_CLASS_BATTERY_CHARGING = "battery_charging"
```

Reply:
Right, what I'm saying is use the battery-charging icon.
flash is overused, and here there are dedicated icons for the purpose

Reply:
Icon set updated